src/vs/workbench/browser/actions/resizePart.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import { TPromise } from 'vs/base/common/winjs.base';
+import nls = require('vs/nls');
+import { Registry } from 'vs/platform/platform';
+import { Action } from 'vs/base/common/actions';
+import { SyncActionDescriptor } from 'vs/platform/actions/common/actions';
+import { IWorkbenchActionRegistry, Extensions } from 'vs/workbench/common/actionRegistry';
+import { IPartService, Parts } from 'vs/workbench/services/part/common/partService';
+import { KeyMod, KeyCode } from 'vs/base/common/keyCodes';
+
+const RESIZE_INCREMENT = 50;
+
+export class ExpandViewAction extends Action {
+
+	public static ID = 'workbench.action.expandView';
+	public static LABEL = nls.localize('expandView', "Expand Current View");
```

Comment:
I suggest to name this differently, e.g. "Increase Current View Size" and `workbench.action.increaseViewSize`